### PR TITLE
Change min-Rust-version to 1.34

### PR DESCRIPTION
**Pull request by lzutao**
cc #12

### CODE REVIEW

**Comment by lzutao** [comment]
bors r=Dylan-DPC,sondr3

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 **Note**: This is a fork of unmaintained `dirs` crate.
 
+There is mid-level sister library, [directories-next].
```
Comment (sondr3) [inline README.md:8]:
```suggestion
There is a mid-level sister library, [directories-next].
```

File: README.md [hunk 2]
```diff
@@ -24,9 +26,12 @@ The library provides the location of these directories by leveraging the mechani
...
 Other platforms are also supported; they use the Linux conventions.
 
-The minimal required version of Rust is 1.36.
+## Minimum Rust version policy
+
+The minimal required version of Rust is `1.34.0`.
 
-It's mid-level sister library, _directories_, is available for Rust ([directories-next]).
+We may bump Rust version in major and minor releases (y in `x.y.z`).
+Changing Rust version will be written in CHANGELOG.
```
Comment (sondr3) [inline README.md:34]:
```suggestion
We may bump the Rust version in major and minor releases (`x`/`y` in `x.y.z`).
Changing the Rust version will be written in the CHANGELOG.
```

File: directories/README.md [hunk 3]
```diff
@@ -5,6 +5,8 @@
 
 **NOTE**: This crate is a fork of unmaintained `directories` crate.
 
+There is low-level sister library, [dirs-next].
```
Comment (sondr3) [inline directories/README.md:8]:
```suggestion
There is a low-level sister library, [dirs-next].
```

File: dirs-sys/README.md [hunk 4]
```diff
@@ -20,8 +20,15 @@ Functions may change or disappear without warning or any kind of deprecation per
...
 Other platforms are also supported; they use the Linux conventions.
 
-The minimal required version of Rust is 1.36 except for Redox, where the minimum Rust version
-depends on the [`redox_users`](https://crates.io/crates/redox_users) crate.
+## Minimum Rust version policy
+
+The minimal required version of Rust is `1.34.0`^.
+
+We may bump Rust version in major and minor releases (y in `x.y.z`).
+Changing Rust version will be written in CHANGELOG.
```
Comment (sondr3) [inline dirs-sys/README.md:28]:
```suggestion
We may bump the Rust version in major and minor releases (`x`/`y` in `x.y.z`).
Changing the Rust version will be written in the CHANGELOG.
```

File: dirs-sys/README.md [hunk 5]
```diff
@@ -20,8 +20,15 @@ Functions may change or disappear without warning or any kind of deprecation per
...
+
+^ Except for Redox, where Rust version depends on the
```
Comment (sondr3) [inline dirs-sys/README.md:30]:
```suggestion
^ Except for Redox, where the Rust version depends on the
```

**Comment by lzutao** [comment]
bors r=Dylan-DPC,sondr3,nmccarty

**Comment by lzutao** [comment]
bors ping

**Comment by lzutao** [comment]
bors retry

**Comment by lzutao** [comment]
bors r=Dylan-DPC,sondr3,nmccarty

**Comment by lzutao** [comment]
bors retry

**Comment by lzutao** [comment]
Manually merge since bors is crashing.

**Pull request merged**